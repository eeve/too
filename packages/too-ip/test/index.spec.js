'use strict';

const ip = require('../src');

const query = ip.create();

it('should query', () => {
  expect(query.btreeSearchSync('114.114.114.114')).toMatchSnapshot();
  expect(query.btreeSearchSync('8.8.8.8')).toMatchSnapshot();
});

it('binarySearch', () => {
  expect(query.binarySearchSync('216.58.200.238')).toMatchSnapshot();
  expect(query.binarySearchSync('204.79.197.200')).toMatchSnapshot();
});
