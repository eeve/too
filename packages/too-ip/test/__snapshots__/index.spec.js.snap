// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`binarySearch 1`] = `
Object {
  "city": 166,
  "region": "美国|0|0|0|0",
}
`;

exports[`binarySearch 2`] = `
Object {
  "city": 166,
  "region": "美国|0|0|0|0",
}
`;

exports[`should query 1`] = `
Object {
  "city": 1015,
  "region": "中国|华东|江苏省|南京市|0",
}
`;

exports[`should query 2`] = `
Object {
  "city": 166,
  "region": "美国|0|0|0|0",
}
`;
